src/pages/AddPage/index.jsx

```diff
@@ -0,0 +1,216 @@
+      setError((prev) => {
+        const newState = [...prev];
+        newState[1] = +value < currentYear;
+        return newState;
+      });
+    }
+
+    if(value.length >= DUE_DATE.UNIT_LENGTH) {
+      dueYearInputRef.current.focus();
+    }
+  };
+
+  const handleOwner = ({target:{value}}) => {
+    if (value.length > OWNER.MAX_LENGTH) return;
+
+    setOwner(value);
+  };
+
+  const handleCvc = ({target:{value}}) => {
+    if (value.length > CVC.UNIT_LENGTH || isNaN(value)) return;
+
+    setCvc(value);
+  };
+
+  const handlePasswordFirst = ({target:{value}}) => {
+    if (value.length > PASSWORD.UNIT_LENGTH || isNaN(value)) return;
+
+    setPassword((prev) => {
+      const newState = [...prev];
+      newState[0] = value;
+      return newState;
+    });
+
+    if(value.length >= 1) {
+      secondPasswordInputRef.current.focus();
+    }
+  }
+
+  const handlePasswordSecond = ({target:{value}}) => {
+    if (value.length > PASSWORD.UNIT_LENGTH || isNaN(value)) return;
+
+    setPassword((prev) => {
+      const newState = [...prev];
+      newState[1] = value;
+      return newState;
+    });
+  }
+
+  const handleSubmit = () => {
+   if(error[0] || error[1]) {
+      alert('만료일을 확인해주세요');
+      return;
+    }
+    alert(`${owner}님의 카드가 등록되었습니다`)
+  }
+
+  return (
+    <PageWrapper>
+      <Header>
+        <Button><Arrow /></Button>
+        <h2>카드 추가</h2>
+      </Header>
+      <CardWrapper>
+        <Card
+          size="small"
+          company="우테코"
+          cardNumbers={cardNumbers}
+          owner={owner || "NAME"}
+          dueMonth={dueDate.month || 'MM'}
+          dueYear={dueDate.year || 'YY'} 
+        />
+      </CardWrapper>
+      <Form 
+        label='카드 번호' 
+        inputInfo={[
+          {type: 'number', id: 'first-card-number', value: cardNumbers[0], ref: cardNumberInputRefs[0]},
+          {type: 'number', id: 'second-card-number', value: cardNumbers[1], ref: cardNumberInputRefs[1]},
+          {type: 'password', id: 'third-card-number', value: cardNumbers[2], ref: cardNumberInputRefs[2]},
+          {type: 'password', id: 'fourth-card-number', value: cardNumbers[3], ref: cardNumberInputRefs[3]}
+        ]}
+        onChange={handleCardNumber}
+        />
+      <Form 
+        label='만료일'
+        size={50}
+        inputInfo={[
+          {type: 'number', id: 'due-month', value: dueDate[0], placeholder: 'MM'},
+          {type: 'number', id: 'due-year', value: dueDate[1], ref: dueYearInputRef, placeholder: 'YY'},
+        ]}
+        error={error[0] || error[1]}
+        onChange={handleDueDate}
+        />
+      <Form 
+        label='카드 소유자 이름 (선택)'
+        countHelper={OWNER.MAX_LENGTH}
+        inputInfo={[
+          {type: 'string', id: 'owner', value: owner, placeholder: '카드에 표시된 이름과 동일하게 입력하세요.'},
+        ]}
+        onChange={handleOwner}
+        />
+      <Form 
+        label='보안 코드(CVC/CVV)'
+        size={30}
+        questionHelper={true}
```

Comment:
prop에 값을 따로 전달하지 않으면 true 로 전달되기 때문에, 보통 true 를 넘길 땐 값을 따로 명시하지 않는 편이에요!
https://reactjs.org/docs/jsx-in-depth.html#props-default-to-true

```suggestion
      <Card isEmpty handleCardAdd={handleCardAdd} />
```